src/Button.js

```diff
@@ -2,18 +2,25 @@
 import React from 'react';
```

Comment:
If you are no longer using semi-colons, you should remove them also from the import statement.

`import React from 'react'`

Reply:
I tend to use semi colons as my IDE comlains about them. I know that when it's compiled it automatically adds them in

Reply:
Thanks @justtoconfirm - removed and re-committed